### ISSUE TITLE
Pip install fails for cattrs==0.7.0 with Python 3

### ISSUE THREAD

**Issue by skycaptain** [Tinche/cattrs#32]
* cattrs version: 0.7.0
* Python version: 3.6
* Operating System: macOS 10.13.4

### Description

Installing latest cattrs via pip fails with python 3.

### What I Did

```
pip install cattrs

Collecting cattrs
  Using cached cattrs-0.7.0-py2.py3-none-any.whl
Collecting typing>=3.5.3 (from cattrs)
  Downloading typing-3.6.4-py3-none-any.whl
Collecting attrs>=17.3 (from cattrs)
  Using cached attrs-17.4.0-py2.py3-none-any.whl
Collecting functools32>=3.2.3 (from cattrs)
  Using cached functools32-3.2.3-2.tar.gz
    Complete output from command python setup.py egg_info:
    This backport is for Python 2.7 only.
```

**Comment by Tinche** [comment]
Hi, should be fixed in 0.8.0. Please verify and close!

**Comment by skycaptain** [comment]
Works. Thanks!
